fetch_secrets.py

```
@@ -0,0 +1,24 @@
+SECRET_KEYS = settings.VAULT_KEYS.split(",")
+
+
+async def fetch_and_append_secrets():
+    await Tortoise.init(config=TORTOISE_ORM)
+    from tortoise import connections
+
+    conn = connections.get("default")
+    columns, rows = await conn.execute_query("SELECT * FROM vault.decrypted_secrets")
+    with open(".env", "a") as env_file:
+        for result in rows:
+            env_file.write(f"{result["name"]}={result["decrypted_secret"]}\n")
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string: unmatched '[' (fetch_secrets, line 20)'

(E0001)